packages/desktop/views/dashboard/settings/views/collectibles/CollectiblesSettings.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+    import { activeProfile, isActiveLedgerProfile } from '@core/profile'
+    import { CollectiblesSettingsRoute } from '@core/router'
+    import features from '@features/features'
+    import { HR } from 'shared/components'
+    import MaxMediaSize from './MaxMediaSize.svelte'
+
+    const settings: {
+        component: unknown
+        childRoute: CollectiblesSettingsRoute
+        requireLogin?: boolean
+        requireLedger?: boolean
+    }[] = [{ component: MaxMediaSize, childRoute: CollectiblesSettingsRoute.MaxMediaSize }]
```

Comment:
Seems this type could be made generic, e.g.
```typescript
interface ISetting<R> {
    component: HTMLElement // perhaps unknown is better
    childRoute: R
    requireLogin?: boolean
    requireLedger?: boolean
}
```

Also, I think `isLedgerSpecific` might be better naming / reasoning since a setting could technically require Ledger _and_ Stronghold for either profile type.

Reply:
This is copied across all the settings pages I have a task to add to GH to refactor these all into on generic component